.github/workflows/analyze.yml

```diff
@@ -9,19 +9,17 @@ jobs:
       - uses: actions/checkout@v2
       - uses: subosito/flutter-action@v1
         with:
-          flutter-version: "2.5.1"
+          channel: stable
```

Comment:
I think it's right to change it like this, but I'm afraid that often unintentional failures will occur.

Reply:
Maybe on new flutter releases?

Reply:
Actually I and @bbrto21 have discussed about this matter before. @bbrto21 Do you remember the conclusion?

Reply:
I agreed to use the stable channel, but it didn't works properly on this repository at that time.(I don't remember exactly why it didn't work)
If there are no more problems, I prefer to use the stable channel.

Reply:
I can't think of any unwanted problem (false alarm) that might be caused by this change.

- A dependency resolution failure (like [this](https://github.com/flutter-tizen/plugins/pull/125#issuecomment-866661681)) may occur if test packages (such as flutter_test) have updated dependencies on new Flutter releases.
- Analysis or formatting rules are added/changed by new versions of Flutter/Dart.

In both cases, the errors should not be suppressed and should be addressed as quick as possible.

Reply:
Okay, I see that CI will detect problems early which will allow us to solve them as soon as possible.